packages/design-system/src/components/enhanced-tooltip.tsx

```diff
@@ -45,6 +45,8 @@ export const EnhancedTooltipProvider = ({
   );
 };
 
+export const useEnhancedTooltipProps = () => useContext(EnhancedTooltipContext);
```

Comment:
wouldn't be better to just export the context? idk

Reply:
Probably it will be just const instead of useContext. Context allows creating Provider. Imo better to wrap